### DISCUSSION TITLE
Pushing up server side validation

### DISCUSSION THREAD

**Discussion by helloadmin** [fenilli/formkit-addon-inertia#16] [Q&A]
I'm feeling this is one of those questions with an incredibly obvious answer.

FormKit has outstanding field validation particularly with displaying the error message and restricting the form from being posted when there is errors.   Laravel additionally has great server side validation.  When a FormKit form passes client side validation and is posted with inertia and then the server side validation fails, I expect there should be a automatic way to have those validation errors show next to the appropriate FormKit fields.

I've looked through the documentation and haven't found references to functionality like this or how to wire it.

Does this exists and whats the basics of wiring it up?

**Comment by fenilli** [comment]
Well that should be the default behaviour if you didn't overwrite the onError callback, I'm also working on a new version of this package where it will look more like inertia API and it will also work with Laravel Precognition.
